druid/src/widget/clip_box.rs

```diff
@@ -315,9 +324,11 @@ impl<T: Data, W: Widget<T>> Widget<T> for ClipBox<T, W> {
         self.port.rect = self.port.rect.with_size(bc.constrain(content_size));
         let new_offset = self.port.clamp_view_origin(self.viewport_origin());
         self.pan_to(new_offset);
+        trace!("Computed sized: {}", self.viewport_size());
```

Comment:
only call this once and reuse the value

Reply:
It's more readable this way, and the function is small enough to let the inliner optimize it.